### ISSUE TITLE
Error: unsupported event: push

### ISSUE THREAD

**Issue by sosedoff** [nektos/act#32]
I'm playing around with an action that runs when a PR is created, however with `act` command i'm getting this error (running `act`):

```
Error: unsupported event: push
```

Example workflow file:

```
workflow "test" {
  on = "pull_request"
  resolves = ["build"]
}

action "build" {
  uses = "./pull-request-action"
}
```

Im not sure why the error refers to `push` event

**Comment by sosedoff** [comment]
Just realized that `act` uses `push` as a default event. When i execute `act pull_request` everything works as expected.

Perhaps `act` could check the workflow file for the event type and run it instead of using a default `push` event.

**Comment by cplee** [comment]
@sosedoff this was intentional, based on the fact that GH actions uses the `push` event as the default as per the [docs](https://developer.github.com/actions/creating-workflows/workflow-configuration-options/#events-supported-in-workflow-files)

![image](https://user-images.githubusercontent.com/2239057/52761947-7ba4f280-2fe3-11e9-93b6-c85af6f4dd1c.png)

**Comment by cplee** [comment]
I'd be open for a PR if you have something else in mind!

**Comment by sosedoff** [comment]
Yeah, i'll experiment a bit on this front. Also, i think i had an older version of `act` installed, once i updated it no longer printed any errors and terminated silently.

**Comment by mcolyer** [comment]
> @sosedoff this was intentional, based on the fact that GH actions uses the `push` event as the default as per the [docs](https://developer.github.com/actions/creating-workflows/workflow-configuration-options/#events-supported-in-workflow-files)

This might have been unintentional on our part. I believe we used the word default here to mean that it's the default event that is selected within the workflow editor. It doesn't really have any significance beyond that. Having support for other events in `act` is definitely encouraged.

**Comment by cplee** [comment]
So `act` does support running other events, but I do like @sosedoff suggestion.  Let's keep this open and implement.

**Comment by sosedoff** [comment]
Ok so i played with workflows a bit more and realized that you can actually have more than one workflow in the same file triggered on the same event (`push`). Im thinking `act` could do the following (to eliminate extra typing):

- Read the workflow file
- If the workflow file contains only a single workflow - detect the event type
- If used did not provide an event we could use the value defined with `on` instead of default `push` 

So if i had a workflow like this:

```
workflow "dummy" {
  on = "pull_request"
  resolves = ["task"]
}
// ...
```

Running `act` would result in equivalent of `act pull_request`. If there are more than one type of events defined in the workflow `act` would use the existing behavior.

**Comment by cplee** [comment]
fixed with #35
